### ISSUE TITLE
undefined method `call' for HamlLint::Tree::NullNode:Class

### ISSUE THREAD

**Issue by CloCkWeRX** [Growstuff/growstuff#1270]
See https://travis-ci.org/Growstuff/growstuff/builds/209222849 blocks #1269 
Maybe https://github.com/brigade/haml-lint/issues/186 ?
```
Analyze with haml-lint.....................................[HamlLint] FAILED
Unexpected output: unable to determine line number or type of error/warning for output:
undefined method `call' for HamlLint::Tree::NullNode:Class
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/tree/root_node.rb:18:in `find'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/tree/root_node.rb:18:in `node_for_line'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/linter/final_newline.rb:9:in `visit_root'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/haml_visitor.rb:40:in `safe_send'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/haml_visitor.rb:19:in `visit'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/linter.rb:28:in `run'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:84:in `block in collect_lints'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:83:in `map'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:83:in `collect_lints'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:119:in `process_file'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:108:in `block in process_files'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:107:in `each'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:107:in `process_files'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:132:in `report'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/runner.rb:23:in `run'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/cli.rb:92:in `scan_for_lints'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/cli.rb:51:in `act_on_options'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/lib/haml_lint/cli.rb:23:in `run'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/gems/haml_lint-0.22.1/bin/haml-lint:7:in `<top (required)>'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/bin/haml-lint:22:in `load'
/home/travis/build/Growstuff/growstuff/vendor/bundle/ruby/2.3.0/bin/haml-lint:22:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli/exec.rb:74:in `load'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli/exec.rb:74:in `kernel_load'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli/exec.rb:27:in `run'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli.rb:335:in `exec'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli.rb:20:in `dispatch'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/cli.rb:11:in `start'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/exe/bundle:32:in `block in <top (required)>'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/lib/bundler/friendly_errors.rb:121:in `with_friendly_errors'
/home/travis/.rvm/gems/ruby-2.3.3@global/gems/bundler-1.14.6/exe/bundle:24:in `<top (required)>'
/home/travis/.rvm/gems/ruby-2.3.3/bin/bundle:22:in `load'
/home/travis/.rvm/gems/ruby-2.3.3/bin/bundle:22:in `<main>'
/home/travis/.rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-2.3.3/bin/ruby_executable_hooks:15:in `<main>'
Report this bug at https://github.com/brigade/haml-lint/issues

```

**Comment by pozorvlak** [comment]
Looks like it, yeah.

**Comment by Br3nda** [comment]
I've got haml-lint updates from depp bot on other open source projects -- and it's causing the build to fail..

Let's lock down haml-lint version in Gemfile until they fix

**Comment by CloCkWeRX** [comment]
Ah, new release just came out :)

**Comment by Br3nda** [comment]
fixed in #1272
